pkg/core/blockchain.go

```diff
@@ -1918,6 +1918,9 @@ func (bc *Blockchain) GetPolicer() blockchainer.Policer {
 
 // GetBaseExecFee return execution price for `NOP`.
 func (bc *Blockchain) GetBaseExecFee() int64 {
+	if bc.persistedHeight == 0 {
+		return interop.DefaultBaseExecFee
```

Comment:
Because we use them to define opcodes prices. Initialisation of native's values happens inside `(m *Management) OnPersist` but we need to know DefaultBaseExecFee and DefaultStoragePrice by this moment. So during genesis block persisting we don't use Policy contract to get them and just use default values.

Reply:
I mean these constants, you're moving them to `interop`.

Reply:
DefaultBaseExecFee was located in the `interop` package before, I didn't move it.

I just moved DefaultStoragePrice from native.Management to native.Policy because management contract does not use it.